Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ruby:2.7.5-alpine3.15 AS middleman
-RUN apk add --no-cache libxml2
-RUN apk add --update --no-cache npm git build-base
+
+RUN apk update && apk add --no-cache libxml2=2.12.7-r3
```

Comment:
apk update is run before installs to ensure you get the latest patched packages.